functorch/functorch/_src/aot_autograd.py

```diff
@@ -282,8 +270,16 @@ def forward(ctx, *flat_tensor_args):
         def backward(ctx, *flat_args):
             contiguous_args = [t.contiguous() for t in flat_args]
             all_args = list(ctx.saved_tensors) + list(contiguous_args)
+            if CompiledFunction.compiled_bw is None:
+                with track_graph_compiling("backward", True):
+                    CompiledFunction.compiled_bw = aot_config.bw_compiler(
+                        bw_module, all_args
+                    )
```

Comment:
Add test? Shouldn't be too bad since we can pass separate compilers for forward and backward

Reply:
`test_grad_context` is already testing this (i.e. this change caused that test to fail haha).

Fixed that.